packages/client/src/config.ts

```diff
@@ -1,6 +1,27 @@
 import { SetupContractConfig } from "@latticexyz/std-client";
 import { Wallet } from "ethers";
+import { foundry } from "@wagmi/chains";
+import type { Chain } from "@wagmi/chains";
+import { latticeTestnet } from "./latticeTestnet";
+import localDeploy from "../../contracts/deploys/31337/latest.json";
+
+const chains: Chain[] = [foundry, latticeTestnet];
```

Comment:
before we do, I just wanted to confirm that wagmi will always _only_ export chains in their package, because `import * as chains` scares me (getting non-chain things)

Reply:
asked here: https://github.com/wagmi-dev/references/discussions/197